conda/plan.py

```diff
@@ -31,10 +30,10 @@
                                 SYMLINK_CONDA)
 
 # Silence pyflakes
-(FETCH, EXTRACT, UNLINK, LINK, RM_EXTRACTED,
-                                RM_FETCHED, PREFIX, PRINT, PROGRESS,
-                                SYMLINK_CONDA)
+(FETCH, EXTRACT, UNLINK, LINK, RM_EXTRACTED, RM_FETCHED, PREFIX, PRINT,
+    PROGRESS, SYMLINK_CONDA)
```

Comment:
I *really* don't like stuff like this.  Doesn't `# NOQA` on the import silence pyflakes?  Let's just do that...